ha/setup/ha_setup.py

```diff
@@ -777,6 +777,13 @@ def process(self):
         Process init command.
         """
         Log.info("Processing init command")
+        env_type = self.get_installation_type().lower()
+        if env_type == const.INSTALLATION_TYPE.HW.value.lower():
+            self._execute.run_cmd(const.PCS_STONITH_ENABLE)
+        elif env_type == const.INSTALLATION_TYPE.VM.value.lower():
+            Log.warn("Stonith configuration not available, detected VM env")
+        else:
+            raise HaConfigException(f"Invalid env detected, {env_type}")
         Log.info("init command is successful")
```

Comment:
update process command

```
            process.run_cmd(f"pcs -f {cib_xml} stonith create {resource_id} fence_ipmilan ipaddr={ipaddr} \
                login={login} passwd={passwd} pcmk_host_list={pcmk_host_list} pcmk_host_check=static-list \
                lanplus=true auth={auth} power_timeout=40 verbose=true \
                op monitor interval=10s meta failure-timeout=15s --group ha_group")
```
It has password 
check process method and pass pass in param as secret

Reply:
Done